<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="b2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>580</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存到proto</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="out1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>720</y>
     <width>1261</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>690</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输出:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>proto对象:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b1">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>520</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>加载到c++</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input3">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>50</y>
     <width>261</width>
     <height>651</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="input2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>871</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>c++对象</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>661</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b3">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>150</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>跳转</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>661</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
